quickevent/app/plugins/Runs/src/eventstatisticswidget.cpp

```diff
@@ -171,30 +171,31 @@ QVariant EventStatisticsModel::value(int row_ix, int column_ix) const
 	if(column_ix == col_timeToCloseMs) {
 		int stage_start_msec = eventPlugin()->stageStartMsec(eventPlugin()->currentStageId());
 		int curr_time_msec = QTime::currentTime().msecsSinceStartOfDay();
-		//int runners_count = value(row_ix, col_runnersCount).toInt();
+		int runners_count = value(row_ix, col_runnersCount).toInt();
 		int runners_finished = value(row_ix, col_runnersFinished).toInt();
-		if(runners_finished == 0) {
-			return QVariant();
+		if(runners_count == runners_finished) {
```

Comment:
if there are less then 3 runners but all finished, timeToClose will be shown corrently as 0